nodes/bm_inventory_r8pac23.json

```diff
@@ -0,0 +1,112 @@
+{
+    "nodes": [
+        {
+            "name": "MOC-R8PAC23U27",
+            "properties": {
+                "capabilities": "iscsi_boot:True"
```

Comment:
Can you set `capabilities` for these nodes to `boot_mode:uefi,iscsi_boot:True`? This will let Ironic know that the node should be booted in the appropriate mode.

Reply:
Sure, done!